Octokit/Models/Request/MergePullRequest.cs

```diff
@@ -11,23 +11,21 @@ namespace Octokit
     [DebuggerDisplay("{DebuggerDisplay,nq}")]
     public class MergePullRequest
     {
-        public MergePullRequest(string message)
-        {
-            Ensure.ArgumentNotNull(message, "message");
-
-            Message = message;
-        }
-
         /// <summary>
         /// The message that will be used for the merge commit (optional)
         /// </summary>
-        public string Message { get; private set; }
+        public string Message { get; set; }
```

Comment:
I'm currently fixing this, but according to the docs here: https://developer.github.com/v3/pulls/#merge-a-pull-request-merge-button

This is _not_ optional and it should be `CommitMessage`. Did you happen to see something that suggests otherwise?